app/models/cart.rb

```diff
@@ -106,7 +106,8 @@ def add_payment_transaction(transaction)
 
     def self.create(*args)
       if FlexCommerceApi.config.order_test_mode
-        super({test: true})
+        args.first&.merge(test: true)
```

Comment:
There may be a better way of doing this. In this scenario: 
```ruby
FlexCommerce::Cart.create({initial_line_items: initial_line_item_attributes})
```
args is passed in as an array. There are scenarios when the array is empty. Although this isn't the cleanest we are overriding the create method is several places so without a big refactor is might be the most pragmatic, plus it'll only effect test orders

https://github.com/shiftcommerce/flex-ruby-gem/blob/621299a0081885e01ddee1e028906519731ab416/lib/flex_commerce_api/base_resource.rb#L39 so we can return an error message.

https://github.com/shiftcommerce/flex-ruby-gem/blob/621299a0081885e01ddee1e028906519731ab416/lib/flex_commerce_api/json_api_client_extension/requestor.rb#L5

https://github.com/shiftcommerce/flex-ruby-gem/blob/038f15de16ed524de581b7049ab1b062e84c140b/lib/flex_commerce_api/json_api_client_extension/has_many_association_proxy.rb#L37

Although not sure what these are doing using the safe operator maybe